src/test/java/uk/gov/ons/census/fwmt/tests/acceptance/steps/CensusSteps.java

```diff
@@ -201,4 +209,41 @@ public void aMessageReceivedFromRmThatFailsToSendToTmAfterAttempts(Integer attem
   public void theErrorIsLoggedViaSPLUNKStoredInA(String queueName) {
     assertEquals(1, queueUtils.getMessageCount(queueName));
   }
+
+  @Given("RM sends a cancel case Household job request with case ID {string}")
+  public void tmHasAnExistingJobWithCaseID(String caseId) throws URISyntaxException, InterruptedException {
+
+    queueUtils.sendToActionFieldQueue(cancelMessage);
+    boolean hasBeenTriggered = gatewayEventMonitor.hasEventTriggered(caseId, CANONICAL_CANCEL_SENT, 10000L);
+    assertThat(hasBeenTriggered).isTrue();
+  }
+
+  @When("the Gateway sends a Cancel Case request to TM with case ID {string}")
+  public void theGatewaySendsACancelCaseRequestToTM(String caseId) {
+    boolean hasBeenTriggered = gatewayEventMonitor.hasEventTriggered(caseId, CANONICAL_CANCEL_RECEIVED, 10000L);
+    assertThat(hasBeenTriggered).isTrue();
+  }
+
+  @Then("a pause datetime of {string} will be assigned to the case with id {string}")
+  public void theCaseWithIdOfWillBeCancelled(String until, String caseId) throws InterruptedException {
+
+    Thread.sleep(1000);
+
+    CasePause casePause = tmMockUtils.getPauseCase(caseId);
+    assertEquals(OffsetDateTime.parse(until), casePause.getUntil());
+  }
+
+  @Given("RM sends a cancel case CSS job request with case ID {string}")
```

Comment:
I'm not sure this mistake could be made by RM? I guess its something to ask Dave/Olu about